Day-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,22 @@
+.medium-btn {
+  background-color: #ffffff;
+  color: #222222;
+  width: 300px;
+  font-size: 45px;
+}
+
+.small-btn {
+  width: 100px;
+  height: 80px;
+  font-size: 45px;
+}
+
+.red {
```

Comment:
Naming it something like `.action-button` is slightly better because you may decide in future that this button should be blue in color. At that point you would have to change the name of the rule to blue and change it in all places you have used this rule.